course7/homeworks/mathias/complex_rpn/main.cpp

```diff
@@ -0,0 +1,30 @@
+#include "rpn_calculator.h"
+#include "complex.h"
+
+#include <iostream>
+#include <string>
+
+int main(int , char** )
+{
+	RpnCalculator calc;
+	std::string line;
+
+	std::cout << "> ";
```

Comment:
Prompt is cool, but useless, and even damaging in batch mode, when commands are read from file. Then the prompt just creates a noise in the output stream. Imagine that you want to use the calculator as part of some automated ipeline, when input is generated and output parsed.